api/base/filters.py

```diff
@@ -177,6 +177,8 @@ def convert_key(self, field_name, field):
         :param basestring field_name: text representation of the field name
         :param rest_framework.fields.Field field: Field instance
         """
+        if getattr(field, 'field', None):
```

Comment:
This was for HideIfRetraction. I wrote this before I decided to `set self.source = field.source` on HideIfRetraction, so it isn't needed as long as I'm setting self.source.

There are many places where `source` is accessed and wrapping these fields with `HideIfRetraction` means I have to implement someway to access the inner source.